TL improvement suggestion (#369)
07th-mod/higurashi-console-arcs: Update/omot_019.txt

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 5, "ps2/05/220500019", 540, TRUE);
		OutputLine(NULL, "「私も質問、いいかしら。",

> 				NULL, "\"I have a question of my own, too, if you don't mind.", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"I have a question of my own, if you don't mind.", Line_WaitForInput);
```

	//rおそらく、息をひそめてやり過ごすつもりなのだろう。…もっとも、何かの心変わりで俺を捕まえて、口封じに出るかもしれないが。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "おそらく、息をひそめてやり過ごすつもりなのだろう。…もっとも、何かの心変わりで俺を捕まえて、口封じに出るかもしれないが。",
				NULL, "They were probably getting away with concealing their breathing. ...But if they decided to change targets, they might capture me to silence me.", Line_Normal);
		ClearMessage();

	//rだが、…今はそんなことを言ってる場合じゃない！
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "だが、…今はそんなことを言ってる場合じゃない！",
				NULL, "But... I had no time to be thinking of them!", Line_Normal);
		ClearMessage();

	//r俺は意を決して、梨花ちゃんの安否を確かめるべく沼の中へ足を踏み入れる。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "俺は意を決して、梨花ちゃんの安否を確かめるべく沼の中へ足を踏み入れる。",

> 				NULL, "I decided to take a step into this swamp in order to confirm whether Rika-chan was safe or not.", Line_Normal);

Reviewer: ```suggestion
				NULL, "I resolutely stepped into this swamp in order to confirm whether Rika-chan was safe or not.", Line_Normal);
```

		OutputLine(NULL, "改めて、自分の身なりに目を向ける。",
				NULL, "I examined myself once more.", Line_Normal);
		ClearMessage();

	//r服は泥だらけになって、手足は切り傷擦り傷、そして汗が混じった土や砂でどろどろになっている…ハズだった。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "服は泥だらけになって、手足は切り傷擦り傷、そして汗が混じった土や砂でどろどろになっている…ハズだった。",
				NULL, "My clothes were muddy, my arms and legs were covered in cuts, and my body was drenched in a mix of dirt, sand, and sweat... or at least, that should've been the case.", Line_Normal);
		ClearMessage();

	//rそれなのに、…服や肌の汚れはそれほどでもなく、さっきまで血がにじんでいた無数の傷も、…消え失せている。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "それなのに、…服や肌の汚れはそれほどでもなく、さっきまで血がにじんでいた無数の傷も、…消え失せている。",
				NULL, "And yet... my clothes and skin didn't look all that dirty, and the countless bloody scratches that were there earlier... had suddenly vanished.", Line_Normal);
		ClearMessage();

	//r…まして、梨花ちゃんはまるで、今まで家にいたかのように汚れひとつない。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "…まして、梨花ちゃんはまるで、今まで家にいたかのように汚れひとつない。",

> 				NULL, "...Furthermore, Rika-chan didn't had a speck of dirt, as if she was home the whole time.", Line_Normal);

Reviewer: ```suggestion
				NULL, "...Furthermore, Rika-chan didn't have a speck of dirt on her, as if she was home the whole time.", Line_Normal);
```

//r赤坂さんはそう言って、ぽんぽんと肩をたたく。さっきの電話の対応をほめてくれているんだろうけど…梨花ちゃんがいなかったら相当、危なかったしな。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "赤坂さんはそう言って、ぽんぽんと肩をたたく。さっきの電話の対応をほめてくれているんだろうけど…梨花ちゃんがいなかったら相当、危なかったしな。",

> 			NULL, "Akasaka-san gave me a pat on the shoulder as he said that. He was probably praising me for what I said on the phone earlier... but I would've made the wrong replies without her help.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Akasaka-san gave me a pat on the shoulder as he said that. He was probably praising me for what I said on the phone earlier... but I would've misspoken without Rika-chan's help.", Line_Normal);
```

			NULL, "\"That's right.", Line_WaitForInput);
	ModPlayVoiceLS(4, 5, "ps3/s14/05/220500044", 256, TRUE);
	OutputLine(NULL, "…考えていることを口に出す癖が直らないうちは、赤坂レベルに到達するのも難しいのですよ」",

> 			NULL, " ...It's hard to reach Akasaka's level unless you can control your own face which speaks of what you're thinking.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...It's hard to reach Akasaka's level unless you can keep your own face from speaking your thoughts.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s14/11/220700036", 256, TRUE);
	OutputLine(NULL, "「ちなみに、東京の公安部資料室は確か、内調別室並みの極秘部門ですよ。",

> 			NULL, "\"And by the way, the reference rooms are classified departments within Public Security at the MPD, and as top secret as the special rooms in CIRO.", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"And by the way, the reference rooms are classified departments within MPD Public Security, and just as top secret as the special rooms in CIRO.", GetGlobalFlag(GLinemodeSp));
```

//r大石さんはそれら全ての話に大仰に驚いていたが、さすがに最後には赤坂さんに静かにしてくださいと言われ、大人しく話を聞いてくれた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大石さんはそれら全ての話に大仰に驚いていたが、さすがに最後には赤坂さんに静かにしてくださいと言われ、大人しく話を聞いてくれた。",

> 			NULL, "Ooishi-san seemed completely blown away by these stories, but Akasaka-san told him to be calm down, so he eventually listened silently.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Ooishi-san seemed completely blown away by these stories, but Akasaka-san urged him to calm down, so he eventually listened silently.", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 11, "ps2/11/220700045", 540, TRUE);
		OutputLine(NULL, "「……こっちから目撃情報を先に提供していたら、やっこさん達は絶対に時間を調整して報告してきたでしょうな」",

> 				NULL, "\"...If they received word from us about the eyewitness accounts, those bastards would've surely altered the time in their reports.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"...If they received word from us about the eyewitness accounts, I'm sure those bastards would've altered the time in their reports.\"", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1035", 256, TRUE);
		OutputLine(NULL, "「ええ…こっちから目撃情報を先に提供していたら、やっこさん達は絶対に時間を調整して報告してきたでしょうな」",

> 				NULL, "\"Right... If they received word from us about the eyewitness accounts, those bastards would've surely altered the time in their reports.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"Right... If they received word from us about the eyewitness accounts, I'm sure those bastards would've altered the time in their reports.\"", Line_Normal);
```

//rそれでも一番の前提『私を疑う』には全然タッチする気もないらしい。…嬉しかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それでも一番の前提『私を疑う』には全然タッチする気もないらしい。…嬉しかった。",

> 			NULL, "Still, He seemed to have no interest about the major hurdle of \"doubting me.\" ...And for that, I'm glad.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Still, he seemed to have no intention of \"doubting me,\" which was the major hurdle. ...And for that, I'm glad.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "そこまでしなければ研究できないのが、雛見沢症候群というものなのだ。……ということは、入江機関が研究を三年で終わらせると宣言した以上、鷹野は個人的に嫌でもどうしようもない、ということになる。",

> 			NULL, "Without that, it's impossible to study Hinamizawa Syndrome. ...Which means if they declared that the Irie Institute's research must end in three years, Takano must have no choice but to understand she won't be able to continue her research.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Without that, it's impossible to study Hinamizawa Syndrome. ...Which means if they declared that the Irie Institute's research must end in three years, no matter how much Takano hates it, there's nothing she can do about it.", Line_Normal);
```

//r今日は昨日の手伝いに職員総出で出払ったこともあり、特別に午後は休みとした。…それは建前で、富竹の失踪と鷹野の死の調査のため、全員を山狗の手伝いに回したのだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "今日は昨日の手伝いに職員総出で出払ったこともあり、特別に午後は休みとした。…それは建前で、富竹の失踪と鷹野の死の調査のため、全員を山狗の手伝いに回したのだ。",

> 			NULL, "All the staff pitched in to help out yesterday, so they were given time off this afternoon as a special courtesy. ...But the truth is that all the Mountain Dogs were busy investigating Tomitake's disappearance and Takano's death, so all of them were assigned to assist the Mountain Dogs today.", Line_Normal);

Reviewer: ```suggestion
			NULL, "All the staff pitched in to help out yesterday, so they were given time off this afternoon as a special courtesy. ...But the truth is, they were all sent to help the Mountain Dogs investigate Tomitake's disappearance and Takano's death.", Line_Normal);
```

//r梨花は、隣の診療室で寝かせてある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "梨花は、隣の診療室で寝かせてある。",
			NULL, "He carried Rika to the exam room next door so that she could rest there.", Line_Normal);
	ClearMessage();

//r…圭一たちと一緒に帰ったと思っていたので、昼過ぎにこの部屋を訪れてちょっと驚いたが、よく考えてみればまだ怪我も完治していない上に、昨晩は演舞で疲労困憊だったはず。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "…圭一たちと一緒に帰ったと思っていたので、昼過ぎにこの部屋を訪れてちょっと驚いたが、よく考えてみればまだ怪我も完治していない上に、昨晩は演舞で疲労困憊だったはず。",

> 			NULL, "...He thought she had headed home with Keiichi and the others last night, so he was surprised to see her here several hours ago, but thinking it over, her injuries still hadn't fully healed and she had to be exhausted from her dance last night.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...He thought she headed home with Keiichi and the others last night, so he was surprised to see her here in the afternoon, but thinking it over, her injuries still hadn't fully healed and she had to be exhausted from her dance last night.", Line_Normal);
```
